<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialogClass</class>
 <widget class="QMainWindow" name="MainDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MEG Video Recording Station</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>540</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>540</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="audioGroupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>91</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Audio</string>
    </property>
    <widget class="QProgressBar" name="levelLeft">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>24</y>
       <width>16</width>
       <height>181</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="levelRight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>24</y>
       <width>16</width>
       <height>181</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="levelTopLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="levelBottomLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="clipLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CLIP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="videoGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Cameras</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="videoVerticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>540</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="markersGroupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>851</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Annotations</string>
    </property>
    <widget class="MarkersWidget" name="markersWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>851</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarkersWidget</class>
   <extends>QWidget</extends>
   <header>markerswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>onStartRec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>993</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>onStopRec()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>993</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>onExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartRec()</slot>
  <slot>onStopRec()</slot>
  <slot>onShutterChanged(int)</slot>
  <slot>onGainChanged(int)</slot>
  <slot>onExit()</slot>
 </slots>
</ui>
